admin/tools/docker/deployment/parallel/run.sh

```diff
@@ -63,13 +63,20 @@ else
     echo "No existing ${CONTAINER_NAME} container on any nodes"
 fi
 
+USE_JEMALLOC='/qserv/stack/stack/current/Linux64/jemalloc/master-g708711ce34/lib/libjemalloc.so'
```

Comment:
I think a better long-term solution would be to extend *setup* to allow an option for setting required environment variables. Something along these lines:

setup --enable=jemalloc